packages/amplify-frontend-javascript/lib/framework-config-mapping.js

```diff
@@ -49,12 +53,41 @@ const defaultConfig = {
   StartCommand: `${npm} run-script start`,
 };
 
-module.exports = {
-  angular: angularConfig,
-  ember: emberConfig,
-  ionic: ionicConfig,
-  react: reactConfig,
-  'react-native': reactNativeConfig,
-  vue: vueConfig,
-  none: defaultConfig,
-};
+function getAngularConfig(projectRoot) {
+  const angularConfigFile = path.join(projectRoot, 'angular.json');
+  const angularProjectConfig = JSONUtilities.readJson(angularConfigFile, { throwIfNotExist: false });
+  let dist = 'dist';
+  if (angularProjectConfig && angularProjectConfig.projects && Object.keys(angularProjectConfig.projects).length) {
+    const { defaultProject } = angularProjectConfig;
+    const projectConfig = angularProjectConfig.projects[defaultProject];
+    dist =
+      (projectConfig &&
+        projectConfig.architect &&
+        projectConfig.architect.build &&
+        projectConfig.architect.build.options &&
+        projectConfig.architect.build.options.outputPath) ||
+      'dist';
+  }
```

Comment:
I think you can use lodash _.get() for all this:
```suggestion
const dist = _.get(angularProjectConfig, ['projects', angularProjectConfig.defaultProject, 'architect', 'build', 'options', 'outputPath'], 'dist');
```